sui_core/src/authority_aggregator.rs

```diff
@@ -1005,8 +1005,8 @@ where
                             // Note: here we aggregate votes by the hash of the effects structure
                             let entry = state
                                 .effects_map
-                                .entry(sha3_hash(&inner_effects.effects))
-                                .or_insert((0usize, inner_effects.effects));
+                                .entry(sha3_hash(&inner_effects.data))
```

Comment:
We should probably not call the sha3_hash directly here but someghing like `.digest()` on the effects. I know this is older code.